uefi/src/main.rs

```diff
@@ -107,7 +122,7 @@ fn main_inner(image: Handle, mut st: SystemTable<Boot>) -> Status {
     )
     .unwrap();
 
-    let framebuffer = init_logger(image, &st, kernel.config);
+    let framebuffer = init_logger(image, &st, config);
```

Comment:
For compatibility, we need to fall back to the framebuffer configuration of the kernel if no min width/height is set in the config file.

```suggestion
    if config.frame_buffer.minimum_framebuffer_height.is_none() {
        config.frame_buffer.minimum_framebuffer_height = kernel.config.frame_buffer.minimum_framebuffer_height;
    }
    if config.frame_buffer.minimum_framebuffer_width.is_none() {
        config.frame_buffer.minimum_framebuffer_width = kernel.config.frame_buffer.minimum_framebuffer_width;
    }
    let framebuffer = init_logger(image, &st, config);
```